src/sql-editor/types.ts

```diff
@@ -203,7 +200,8 @@ export enum SuggestionKind {
   BoolValues = 'boolValues',
   NullValue = 'nullValue',
   NotKeyword = 'notKeyword',
-  TemplateVariables = "templateVariables",
+  TemplateVariables = 'templateVariables',
+  StarWildCard = 'StarWildCard',
```

Comment:
```suggestion
  StarWildCard = 'starWildCard',
```

nit: match the casing with the other enum values.

Reply:
thanks!